vite.config.ts

```diff
@@ -10,6 +10,10 @@ export default defineConfig({
   test: {
     environment: 'jsdom',
     exclude: [...configDefaults.exclude, 'e2e'],
+    coverage: {
+      provider: 'v8',
+      reporter: ['text', 'html', 'clover', 'json', 'lcov'],
```

Comment:
maybe unnecessary to produce all the reports -- lcov (or lcovonly since the report is visualized in Sonar) could be sufficient